api/tests/routes/pro/patch_offerer_address_test.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+
+from pcapi.core.offerers import models as offerers_models
+import pcapi.core.offerers.factories as offerers_factories
+from pcapi.core.testing import assert_num_queries
+import pcapi.core.users.factories as users_factories
+
+
+class PatchOffererAddressTest:
+    @pytest.mark.usefixtures("db_session")
+    def test_user_can_change_offerer_address_label(self, client):
+        pro_user = users_factories.ProFactory()
+        offerer = offerers_factories.OffererFactory()
+        offerer_id = offerer.id
+        offerers_factories.UserOffererFactory(user=pro_user, offerer=offerer)
+        offerer_address = offerers_factories.OffererAddressFactory(offerer=offerer)
+        offerer_address_id = offerer_address.id
+        expected_data = {"label": "New label"}
+
+        http_client = client.with_session_auth(pro_user.email)
+        # Fetch the session
+        # Fetch the user
+        # Check permission
+        # Check offerer address exists
+        # Update offerer address
+        with assert_num_queries(5):
+            response = http_client.patch(f"/offerers/{offerer_id}/address/{offerer_address_id}", json=expected_data)
+            assert response.status_code == 204
+
+        offerer_address = offerers_models.OffererAddress.query.one()
+        assert offerer_address.label == expected_data["label"]
+
+    @pytest.mark.usefixtures("db_session")
+    def test_user_cant_change_offerer_address_on_foreign_offerer(self, client):
+        pro_user = users_factories.ProFactory()
+        offerer = offerers_factories.OffererFactory()
+        offerer_2 = offerers_factories.OffererFactory()
+        offerer_2_id = offerer_2.id
+        offerers_factories.UserOffererFactory(user=pro_user, offerer=offerer)
+        offerer_address = offerers_factories.OffererAddressFactory(offerer=offerer_2)
+        offerer_address_id = offerer_address.id
+        new_label = {"label": "New label"}
+
+        http_client = client.with_session_auth(pro_user.email)
+
+        # Fetch the session
+        # Fetch the user
+        # Check permission
+        # rollback
+        with assert_num_queries(4):
+            response = http_client.patch(f"/offerers/{offerer_2_id}/address/{offerer_address_id}", json=new_label)
+            assert response.status_code == 403
+        offerer_address = offerers_models.OffererAddress.query.one()
+        assert offerer_address.label != new_label["label"]
+
+    @pytest.mark.usefixtures("db_session")
+    def test_user_cant_change_offerer_address_he_doesnt_own(self, client):
+        pro_user = users_factories.ProFactory()
+        offerer = offerers_factories.OffererFactory()
+        offerer_1_id = offerer.id
```

Comment:
Détail : Je m'y perds dans le nommage
```suggestion
        offerer_1 = offerers_factories.OffererFactory()
        offerer_1_id = offerer_1.id
```